mobile/src/reducers/organization/departments.reducer.ts

```diff
@@ -0,0 +1,24 @@
+import { ActionsObservable } from 'redux-observable';
+import { LoadDepartments, loadDepartmentsFinished, DepartmentsActions } from './departments.action';
+import { ajaxGetJSON } from 'rxjs/observable/dom/AjaxObservable';
+import { deserializeArray } from 'santee-dcts/src/deserializer';
+import { Department } from './department.model';
+import { Reducer } from 'redux';
+
+export const loadDepartmentsEpic$ = (action$: ActionsObservable<LoadDepartments>) =>
+action$.ofType('LOAD-DEPARTMENTS')
+    .first()
+    .flatMap(x => ajaxGetJSON('http://localhost:5000/api/departments'))
```

Comment:
Do you mind to put this out into some config file?

Reply:
Yes, I forgot to add todo for this :) This will be a separate task as it requires a bit of research

Reply:
see #4 task